### ISSUE TITLE
Annotation on Cvat

### ISSUE THREAD

**Issue by 7777-web** [openvinotoolkit/datumaro#543]
Please I need help.

It’s my first time annotating images using cvat. I’ve annotated them but these are the issues I have

1. The images don’t have distinct colors
2. 2. When j try exporting it exports as dataset and I need mask png.

How do I get the mask png? Thanks.

**Comment by zhiltsov-max** [comment]
Hi. I'm not sure I understand the problem, but try to read these guides:

https://openvinotoolkit.github.io/cvat/docs/manual/advanced/annotation-with-polygons/creating-mask/
https://openvinotoolkit.github.io/cvat/docs/manual/advanced/formats/format-voc/#segmentation-mask-export

If you need png masks, export the task or project in the "Segmentation Mask" format.

**Comment by 7777-web** [comment]
What I mean is, how do I download images and not data set. When I download
via segmentation I don’t see my annotated images. I just see scripts.

**Comment by zhiltsov-max** [comment]
https://openvinotoolkit.github.io/cvat/docs/manual/advanced/downloading-annotations/

Do you use the "save images" checkbox?
![image](https://user-images.githubusercontent.com/13832349/141128555-ac6f5c5c-a227-4331-8f15-4b9c030bc663.png)

**Comment by 7777-web** [comment]
Yes I did, but my “export format” doesn’t have segmentation mask in it
except I open the job link and export from there, and even if I do, it
doesn’t show anything.

It gives me this instead

**Comment by zhiltsov-max** [comment]
Could you provide any screenshots to explain the situation better? Which CVAT version are you using? The segmentation mask format was added in [v0.5](https://github.com/openvinotoolkit/cvat/releases/tag/v0.5.0) as mask and then renamed.
